README.md

```diff
@@ -0,0 +1,103 @@
+## Generate a new Maven project ##
+
+- Run below command from Command line tool
+
+```text
+mvn archetype:generate -DartifactId=selenium-learning -DgroupId=com.mytest -DarchetypeGroupId=org.apache.maven.archetypes -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4
+```
+
+- Run below command to download Maven Dependencies, compile, and build project
+
+```text
+mvn clean package -e
+```
+OR,
+
+```text
+mvn clean install -e
+```
+**NOTE:** Use argument `-e` to console error messages while building the project 
+
+- Run below command to run unit tests against compile source code using available testing framework
+
+```text
+mvn test
+```
+
+## Add required dependency for Maven project ##
+
+- Update/Modify the deppendencies in `pom.xml`. Refer [Maven Central Repository](https://central.sonatype.com/) to know about the latest version of dependencies.
+
+1. Add Selenium Dependency
+
+```xml
+<dependency>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-java</artifactId>
+      <version>4.18.1</version>
+</dependency>
+```
```

Comment:
**Issue**: Adding a single Selenium dependency without discussing the need for specific browser drivers might lead to confusion for beginners. <br> **Fix**: Expand the section on adding Selenium dependency to include details on why browser-specific drivers are necessary and how to add them. <br> **Code Suggestion**: 
 ```
 To use Selenium for your testing, you need to add the Selenium Java dependency to your `pom.xml`. This allows you to control browsers with your tests. Additionally, for each browser you plan to use, you need a corresponding driver, which acts as a bridge between your tests and the browser. For example, to test on Chrome, you would need the ChromeDriver.
 ```